src/pages/dar_collection_review/ApplicationInformation.js

```diff
@@ -92,11 +138,71 @@ export default function ApplicationInformation(props) {
       !isLoading ? div({className: 'non-technical-summary-subheader', style: styles.title}, ["Non-Technical Summary"])
         : div({className: 'text-placeholder', key: 'non-technical-summary-title-placeholder', style: {height: '4rem', width: '20%', marginBottom: '2rem'}}),
       div({className: 'non-technical-summary-container'}, [
-        !isLoading ? div({className: 'non-technical-summary-textbox', style: styles.nonTechSummaryText}, [nonTechSummary])
+        !isLoading ? div({className: 'non-technical-summary-textbox', style: styles.textBox}, [nonTechSummary])
           : div({className: 'text-placeholder', key: 'non-technical-summary-placeholder', style: { height: '18rem',
             width: '100%',
           }})
-      ])
+      ]),
+      div({className: 'application-details-container', style: { margin: '2.5rem 0'}}, [
+        div({className: 'applicant-details-subheader', style: styles.title}, ["Application Details"]),
+        // figure out what data needs to be visible and save that data in an array
+        // using generateLabelSpanContents to generate the value for each element in the array
+          // todo: do I include the if logic in here or within the function?
+        DynamicRowGeneration(2, [
+            generateLabelSpanContents('External Collaborators', 'external-collaborators', externalCollaborators, isLoading),
```

Comment:
Don't generate the labels and spans from the start. You'll need to figure out what fields to show or not show. For example, if `localComputing` and `cloudComputing` are `undefined` or an empty string then there's no reason to generate templates for them at all.

Look at the values directly and see if it makes sense to render them out. If it does, you can add them to an array. Once you've made your array you can use `generateLabelSpanContents` on each element to get your templates, then chunk them afterwards.